src/components/DateInput.jsx

```diff
@@ -56,6 +58,7 @@ const defaultProps = {
   placeholder: 'Select Date',
   displayValue: '',
   screenReaderMessage: '',
+  ariaLabelledBy: null,
```

Comment:
this may need to be undefined? (same with all the added defaults)

Reply:
That would also make handling testing easier